index.bs

```diff
@@ -5401,6 +5405,41 @@ authenticator=] and [=client device=] SHOULD work together to ensure that the ex
 only to the operating system user that created that [=platform credential=].
 
 
+## Username Enumeration ## {#sctn-username-enumeration}
+
+While initiating a [=registration=] or [=authentication=] [=ceremony=], there is a risk that the [=[WRP]=] might leak sensitive
+information about its registered users. For example, if a [=[RP]=] uses e-mail addresses as usernames and a user attempts to
+initiate an [=authentication=] [=ceremony=] for "alex.p.mueller@example.com" and the [=[RP]=] responds with a failure, but then
+successfully initiates an [=authentication ceremony=] for "j.doe@example.com", then the user can conclude that "j.doe@example.com"
+is registered and "alex.p.mueller@example.com" is not. The [=[RP]=] has thus leaked the possibly sensitive information that
+"j.doe@example.com" has an account at this [=[RP]=].
+
+The following is a non-normative, non-exhaustive list of measures the [=[RP]=] MAY implement to mitigate or prevent information
+leak due to such an attack:
+
+- When initiating an [=authentication ceremony=], generate and return a valid {{PublicKeyCredentialRequestOptions}} object
+    even if the supplied username is not registered. The {{PublicKeyCredentialRequestOptions/allowCredentials}} member may be
+    populated with random values.
+
+- When verifying an {{AuthenticatorAssertionResponse}} response from the [=authenticator=], make it indistinguishable whether
+    verification failed because the signature is invalid or because no such user or credential is registered.
+
+- If the [=[RP]=] uses [=[RP]=]-specific usernames as user identifiers:
```

Comment:
Yes, what I wanted to express with that was that it's a username that's not shared between RPs, so that it's less likely that one person has the same username on different sites and also less likely that the same username on different sites belongs to teh same person.

As for the term "user identifiers", perhaps I could rewrite "as user identifiers" to "to identify users". I don't think I actually need the term as such.